### DISCUSSION TITLE
You were last verified as a student on . (blank)

### DISCUSSION THREAD

**Discussion by GeorgeSedhom0** [community/community#128748] [GitHub Education] [answered]
### Hi everyone,

I have just re-applied for the student pack again, and did that
i got a rejection email with some steps before applying again
![image](https://github.com/community/community/assets/90509830/0397af9f-4296-4ca5-b0a1-081f44df3f8f)
with this email right after it also confirming my application was rejected
![image](https://github.com/community/community/assets/90509830/112bd795-1cac-4583-be93-6ea2a7d5fd2c)
But when i try to apply again after making some changes and getting a better detailed image of the document, i see this error
![image](https://github.com/community/community/assets/90509830/9d4413c3-aa60-4ea1-b4d0-c0ac67852a1d)
i don't know where to go from here, any suggestions are very appreciated

**Comment by samus-aran** [comment, marked as answer]
Hi @GeorgeSedhom0 

We can see you have a currently Approved submission so you should just need to wait for the 72 hours period has passed to continue your academic benefits. 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You were last verified as a student on "blank".](https://github.com/orgs/community/discussions/111352#last-verified-on-blank)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
